dwolla_v2.gemspec

```diff
@@ -24,6 +24,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rspec", "~> 3.0"
   spec.add_development_dependency "webmock", "~> 1.22"
 
+  spec.add_dependency "public_suffix", "~> 1.4.6"
```

Comment:
Hey @machadolab,

We originally added this because `1.x` is required to support Ruby `< 2`, but weren't aware of the issues it causes with other gems. I just opened a [branch](https://github.com/Dwolla/dwolla-v2-ruby/pull/20) that removes the specific version requirement from the gemspec along with a couple other small changes (see the changelog at the bottom of the README).

You can try it out by updating your `Gemfile`:

```ruby
gem 'dwolla_v2', github: 'dwolla/dwolla-v2-ruby', branch: '2.0-dev'
```

Thanks for the heads up!